src/main/java/com/blackducksoftware/integration/jira/config/controller/BlackDuckConfigActions.java

```diff
@@ -52,9 +46,6 @@ public BlackDuckServerConfigSerializable getStoredBlackDuckConfig(final PluginSe
         final String blackDuckUrl = getValue(settings, BlackDuckConfigKeys.CONFIG_BLACKDUCK_URL);
         logger.debug(String.format("Returning Black Duck details for %s", blackDuckUrl));
         final String apiToken = getValue(settings, BlackDuckConfigKeys.CONFIG_BLACKDUCK_API_TOKEN);
-        final String username = getValue(settings, BlackDuckConfigKeys.CONFIG_BLACKDUCK_USER);
-        final String password = getValue(settings, BlackDuckConfigKeys.CONFIG_BLACKDUCK_PASS);
-        final String passwordLength = getValue(settings, BlackDuckConfigKeys.CONFIG_BLACKDUCK_PASS_LENGTH);
         final String timeout = getValue(settings, BlackDuckConfigKeys.CONFIG_BLACKDUCK_TIMEOUT);
         final String trustCert = getValue(settings, BlackDuckConfigKeys.CONFIG_BLACKDUCK_TRUST_CERT);
         final String proxyHost = getValue(settings, BlackDuckConfigKeys.CONFIG_PROXY_HOST);
```

Comment:
We probably want to remove all use of noProxyHosts, since it is no longer supported.